API.md

```diff
@@ -1,12 +1,120 @@
 # Authmosphere API Documentation
 
+## Token Cache
+
+Class to request and cache tokens on client-side.
+
+#### Usage
+
+```typescript
+
+// TokenCacheOptions
+const cacheOptions = {
+  cacheConfig: {
+    percentageLeft: 0.75
+  },
+  logger: someExternalLogger
+};
+
+const oAuthConfig = {
+  grantType: OAuthGrantType.CLIENT_CREDENTIALS_GRANT,
+  accessTokenEndpoint: 'https://....',
+  credentialsDir: './credentialsDir'
+};
+
+const tokenConfig = {
+  'service-foo': ['foo.read', 'foo.write'],
+  'service-bar': ['bar.read']
+};
+
+// create a new cache
+const tokenCache = new TokenCache(tokenConfig, oAuthConfig, cacheOptions);
+
+// Get a token from cache
+tokenCache
+  .get('service-foo') // Needs to match with a key from 'tokenConfig'
+  .then((token: Token) => {
+    console.log(token.access_token);
```

Comment:
Maybe we should not do the one thing you definitely should not do with a token in our example ;)
In other examples we used a comment like:

`// ...use the token...`

Reply:
✅